app/crud.py

```
@@ -0,0 +1,152 @@
+    """
+    try:
+        db.add(todo)
+        await db.commit()
+        await db.refresh(todo)
+        return todo
+    except Exception as e:
+        await db.rollback()
+        raise HTTPException(status_code=500, detail=f"Error creating ToDo: {e}")
+
+
+async def get_todos(db: AsyncSession):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_todos`

(ANN201)